packages/CodingStandard/src/Tokenizer/ClassTokensAnalyzer.php

```diff
@@ -48,22 +48,31 @@ public function getClassEnd(): int
     }
 
     /**
+     * @return mixed[]
+     */
+    public function getProperties(): array
+    {
+        return $this->filterClassyTokens($this->tokensAnalyzer->getClassyElements(), ['property']);
```

Comment:
@vlastavesely I found this cool native method `getClassyElements()` that automatically finds all constants, properties and methods.
So I've delegated responsibility out and that simplified the code a lot.

Reply:
Amazing 👍